src/routing/http-routing-v1.md

```diff
@@ -318,82 +355,104 @@ the case, the field MUST be ignored.
 
 :::
 
-### Legacy Schemas
+### Announcement Schema
 
-Legacy schemas include `ID` and optional `Addrs` list just like
-the [`peer` schema](#peer-schema) does.
+The `announcement` schema can be used in `PUT` operations to announce content providers or peer routing information.
 
-These schemas are deprecated and SHOULD be replaced with `peer` over time, but
-MAY be returned by some legacy endpoints. In such case, a client MAY parse
-them the same way as the `peer` schema.
-
-#### Bitswap Schema
-
-A legacy schema used by some routers to indicate a peer supports retrieval over
-the `/ipfs/bitswap[/*]` libp2p protocol.
 
 ```json
-{
-  "Protocol": "transport-bitswap",
-  "Schema": "bitswap",
-  "ID": "bafz...",
-  "Addrs": ["/ip4/..."]
-}
+  {
+    "Schema": "announcement",
+    "Payload": {
+      "CID": ["cid1", "cid2", ...],
+      "Scope": "block",
+      "Timestamp": "YYYY-MM-DDT23:59:59Z",
+      "TTL": 0,
+      "ID": "12D3K...",
+      "Addrs": ["/ip4/...", ...],
+      "Protocols": ["foo", ...],
+      "Metadata": "mbase64-blob",
+    },
+    "Signature": "mbase64-signature"
```

Comment:
signatures over blocks of announcements end up being awkward, because the intermediate server can't provide a compact provider record onwards that a subsequent node can verify.

I guess the use of double-hashed content routing records gives us a better base for partial authenticity of individual records.


In what cases are we hoping for transient verifiability of blocks of http put routing records?

Reply:
I've pushed 30a5e3bf3603f9c3207668254a672704d1c3c2e0 which removes the concept of singing more than one CID.